table/tables/tables.go

```diff
@@ -673,6 +661,27 @@ func (t *Table) IterRecords(ctx context.Context, startKey kv.Key, cols []*table.
 	return nil
 }
 
+func GetColDefaultValue(ctx context.Context, col *table.Column, defaultVals []types.Datum) (
```

Comment:
If implement a `GetAllColDefaultValues`, it will be an interface and its arguments are hard to understand. I tried and thought it was not very good. @hanfei1991